tasks/transform_conformance/tests/babel-plugin-transform-typescript/test/fixtures/preserve-import-=/output.js

```diff
@@ -0,0 +1,3 @@
+import nsa from "./module-a";
+var Foo = nsa.bar;
```

Comment:
The `Foo` is not removed, it is a typing.

Reply:
`only_remove_type_imports` is enabled, so it only removes type-only imports. This option is the same as [verbatimModuleSyntax] (https://www.typescriptlang.org/tsconfig/#verbatimModuleSyntax) in `TypeScript`.

Reply:
See https://www.typescriptlang.org/play/?noLib=true&skipLibCheck=true&target=99&jsx=1&verbatimModuleSyntax=true#code/JYWwDg9gTgLgBAOwM4EM4DMoRHARAOgHoQIATAVwBsBTAWhVwG4AoUSWOAMQgjgF5EqfACMUUFswDGEZPHQ8AXFx784ABkZA

Reply:
Ok. I get you. It shoulde be remove if `only_remove_type_imports` is enabled ,right?

Reply:
Yes!